include/sapi/marshal.h

```diff
@@ -414,6 +414,20 @@ TPM2B_PUBLIC_KEY_RSA_Unmarshal (
     TPM2B_PUBLIC_KEY_RSA *dest);
 
 TSS2_RC
+TPM2B_PRIVATE_KEY_RSA_Marshal (
```

Comment:
Should these TPM2B_PRIVATE_KEY_RSA* function prototypes be in the commit dc58ee361125b792571fa656f1182c8bfd51af67?